### PR TITLE
Slack Connector with Fixed PR

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+If you don’t know how to use a properties file, see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders].
+
+Before you try the examples, access Anypoint Studio (Studio) and verify that the *Mule Palette* view displays entries for Slack Connector. If not, follow the instructions in xref:slack-connector-studio.adoc#add-connector-to-project[Add the Connector to Your Mule Project].
+
+== Creating a Channel
+
+This example shows how to create a public Slack channel with a user-specified name.
+
+The following screenshot shows app flow for this example:
```
Comment (valkyrie69) [inline slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc]:
```suggestion
The following screenshot shows the app flow for this example:
```

File: slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+. Accept the defaults.
+. On the Listener properties window, set the *Path* field value to `/create-channel`.
+
+=== Add the First Transform Message Component
+
+This *Transform Message* component takes the channel name as input and then converts the name to JSON format:
+
+. In the *Mule Palette* view, select *Core* and drag the *Transform Message* component to the right of *Listener*.
+. Change the name of the *Transform Message* component to `Channel Details`.
+. Click the *Transform Message* component, add the channel name to the output, and set the output format to application/json:
```
Comment (valkyrie69) [inline slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc]:
```suggestion
. Click the *Transform Message* component, add the channel name to the output, and set the output format to `application/json`:
```

File: slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+. Enter the following values:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Consumer Key | `${consumer.key}`
+|Consumer Secret| `${consumer.secret}`
+|Authorization url | `\https://slack.com/oauth/authorize`
+|Access token url | `\https://slack.com/api/oauth.access`
+|Scopes | `channels, enter write groups:write im:write mpim:write users:read.email`
```
Comment (valkyrie69) [inline slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc]: For Scopes, is the channels extraneous? Should it just be Enter the write groups: groups:write, im:write, mpim:write..." etc ?

File: slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+. In the *Mule Palette* view, select *Core* and then select the *Logger* component.
+. Configure the *Logger* component.
+. Save the project.
+
+=== Run the App
+
+To run the app:
+
+. Start the Mule app.
+. Send as POST request to `\http://localhost:8081/account?name="your channel name"``.
```
Comment (valkyrie69) [inline slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc]:
```suggestion
. Send a POST request to `\http://localhost:8081/account?name="your channel name"``.
```

File: slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+		<logger level="INFO" doc:name="Result" message="#[payload]"/>
+	</flow>
+</mule>
+----
+
+== Posting a Message to a Channel
+
+This example shows how to post a message to a public, private, or direct message/IM channel.
+
+The following screenshot shows app flow for this example:
```
Comment (valkyrie69) [inline slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc]:
```suggestion
The following screenshot shows the app flow for this example:
```

File: slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
++
+Specifies the target channel and provides the message content for the Send Message operation
++
+* Send Message operation
++
+Connects with Slack and posts the message to the specified channel
++
+* Second Transform Message component
++
+Outputs the results of the `Send Message` operation in JSON format
```
Comment (valkyrie69) [inline slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc]: Elsewhere you didn't format the operation name (just used title case).

Comment (valkyrie69) [inline slack/1.0/modules/ROOT/pages/slack-connector-examples.adoc]:
```suggestion
Outputs the results of the Send Message operation in JSON format
```

File: slack/1.0/modules/ROOT/pages/slack-connector-reference.adoc [hunk 7]
```diff
@@ -0,0 +1,13146 @@
...
+** MINUTES
+** HOURS
+** DAYS |  Time unit that qualifies the Response Timeout. |  MINUTES | 
+| Streaming Type a| Enumeration, one of:
+
+** AUTO
+** ALWAYS
+** NEVER |  Defines whether to send the request using streaming. If the value is set to `AUTO` the best strategy is automatically determined based on the content of the request. |  AUTO | 
+| Name a| String |  The identifier of this element used to reference it in other components |  | *x*
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform will expire the instance at the exact moment that it becomes eligible. The runtime will actually purge the instances when it sees it fit. |  |
```
Comment (valkyrie69) [inline slack/1.0/modules/ROOT/pages/slack-connector-reference.adoc]: This is the old Expiration Policy description. We have a proposed new one on our spreadsheet (not that it's much better).

**Review by valkyrie69 — DISMISSED** [review]

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**